detector/muonSystem/muonSystemMuRWELL_o1_v01.cpp

```diff
@@ -616,7 +616,7 @@ static dd4hep::Ref_t createmuonSystemMuRWELL_o1_v01(dd4hep::Detector& lcdd,
     endcapTrans = dd4hep::Position(0., 0., endcapType * endcapOffset);
     endcapPhys = detectorVolume.placeVolume(endcapVolume, dd4hep::Transform3D(dd4hep::RotationZ(0.), endcapTrans));
     endcapPhys.addPhysVolID("type", endcapType);
-    EndcapDE = dd4hep::DetElement(detElement, "MSEndcapDE" + endcapType , endcapType);
+    EndcapDE = dd4hep::DetElement(detElement, "MSEndcapDE" + std::to_string(endcapType) , endcapType);
```

Comment:
In this case `endcapType` will be `-1` (see https://github.com/key4hep/k4geo/blob/master/detector/muonSystem/muonSystemMuRWELL_o1_v01.cpp#L610)  so it will look like "MSEndcapDE-1" and you can only know it's -1 if you see it with other numbers like 0, 1.

Reply:
Indeed. It is not a big deal but this would slightly help
```suggestion
    EndcapDE = dd4hep::DetElement(detElement, "MSEndcapDE_" + std::to_string(endcapType) , endcapType);
```

the other namings could be homogenized as well, e.g. https://github.com/key4hep/k4geo/blob/a63787154f6c185b715c7bb998fb0e1ad08a2fc3/detector/muonSystem/muonSystemMuRWELL_o1_v01.cpp#L634 but we could let @mahmoudali2 choose a naming convention and homogenize. @mahmoudali2 , do you want to do it in a separated PR ?